tests/io/vasp/test_sets.py

```diff
@@ -107,21 +107,21 @@ def test_sets_changed(self):
             with open(input_set) as file:
                 text = file.read().encode("utf-8")
                 name = os.path.basename(input_set)
-                hashes[name] = hashlib.sha1(text).hexdigest()
+                hashes[name] = hashlib.sha256(text).hexdigest()
 
         known_hashes = {
-            "MatPESStaticSet.yaml": "8edecff2bbd1932c53159f56a8e6340e900aaa2f",
-            "MITRelaxSet.yaml": "1a0970f8cad9417ec810f7ab349dc854eaa67010",
-            "MPAbsorptionSet.yaml": "5931e1cb3cf8ba809b3d4f4a5960d728c682adf1",
-            "MPHSERelaxSet.yaml": "0d0d96a620461071cfd416ec9d5d6a8d2dfd0855",
-            "MPRelaxSet.yaml": "f2949cdc5dc8cd0bee6d39a5df0d6a6b7c144821",
-            "MPSCANRelaxSet.yaml": "167668225129002b49dc3550c04659869b9b9e47",
-            "MVLGWSet.yaml": "104ae93c3b3be19a13b0ee46ebdd0f40ceb96597",
-            "MVLRelax52Set.yaml": "4cfc6b1bd0548e45da3bde4a9c65b3249da13ecd",
-            "PBE54Base.yaml": "ec317781a7f344beb54c17a228db790c0eb49282",
-            "PBE64Base.yaml": "480c41c2448cb25706181de268090618e282b264",
-            "VASPIncarBase.yaml": "19762515f8deefb970f2968fca48a0d67f7964d4",
-            "vdW_parameters.yaml": "04bb09bb563d159565bcceac6a11e8bdf0152b79",
+            "MVLGWSet.yaml": "eba4564a18b99494a08ab6fdbe5364e7212b5992c7a9ef109001ce314a5b33db",
```

Comment:
pinging @mkhorton to let you know @DanielYang59 switched to a new hash algo for the VASP input sets (presumably recommended by some `ruff` rule?) and updated the `known_hashes` accordingly

Reply:
Yes https://github.com/materialsproject/pymatgen/pull/3892#discussion_r1650885177